src/connectivities.jl

```diff
@@ -48,3 +48,27 @@ end
 function Base.show(io::IO, c::Connectivity{PL}) where {PL}
   print(io, "$PL$(c.list)")
 end
+
+"""
+Convention used to reconstruct the faces of a polytope
+from a list of vertices.
+"""
+abstract type OrderingConvention end
```

Comment:
We need a new file `src/conventions.jl` to place this abstract type. This file will also include the `src/conventions/gmsh.jl`

Reply:
Ok, I was wondering if it was necessary to have a file with very few lines. But that makes sense.